src/qml/main.qml

```diff
@@ -51,5 +51,12 @@ ApplicationWindow {
         function onLoginSuccess() {
             stackview.push(chat)
         }
+        function onShowError(title, errorMessage) {
+            var popupComponent = Qt.createComponent("qrc:/PopupMessage.qml")
+            var popupMessageError = popupComponent.createObject(stackview);
+            popupMessageError.open()
+            popupMessageError.title = qsTr(title)
```

Comment:
I don't know if this can work for translations...

Reply:
How can i improve it?

Reply:
Try to translate and see if it works... For now we can also stay only with english

Reply:
Oh I think I understand the problem. I used shortsightedly qsTr.